Bestiary/Descent into Avernus.xml

```diff
@@ -0,0 +1,1520 @@
+		<type>Fiend (demon)</type>
+		<alignment>chaotic evil</alignment>
+		<ac>13 (natural armor)</ac>
+		<hp>10 (3d4+3)</hp>
+		<speed>30 ft., fly 30 ft. (bad flier)</speed>
+		<str>6</str>
+		<dex>14</dex>
+		<con>13</con>
+		<int>4</int>
+		<wis>9</wis>
+		<cha>5</cha>
+		<resist>cold, fire, lightning</resist>
+		<immune>poison</immune>
+		<conditionImmune>blinded, poisoned</conditionImmune>
+		<senses>blindsight 30 ft. (blind beyond this radius)</senses>
+		<passive>9</passive>
+		<languages>Understands Abyssal, but can't speak</languages>
+		<cr>1/4</cr>
+		<trait>
+			<name>Bad Flier</name>
+			<text>The abyssal chicken falls at the end of its turn if it's airborne and the only thing holding it aloft is its flying speed</text>
+		</trait>
+		<action>
+			<name>Multiattack</name>
+			<text>The abyssal chicken makes two attacks: one with its bite and one with its claws.</text>
+		</action>
+		<action>
+			<name>Bite</name>
+			<text>Melee Weapon Attack: +4 to hit, reach 5 ft., one creature. Hit: 4 (1d4 + 2) damage.</text>
+			<attack>|4|1d4+2</attack>
+		</action>
+		<action>
+			<name>Claws</name>
+			<text>Melee Weapon Attack: +4 to hit, reach 5 ft., one target. Hit: 5 (1d + 2) damage.</text>
+			<attack>|4|1d6+2</attack>
+		</action>
+		<environment>Any</environment>
+		<description>Source: Baldur's Gate: Descent Into Avernus</description>
+	</monster>
+	<monster>
+		<name>Fiendish Flesh Golem</name>
+		<size>L</size>
+		<type>construct</type>
+		<alignment>unaligned</alignment>
+		<ac>12 (natural armor)</ac>
+		<hp>210 (20d10+100)</hp>
+		<speed>30 ft., fly 30 ft. (hover)</speed>
+		<str>20</str>
+		<dex>9</dex>
+		<con>20</con>
+		<int>7</int>
+		<wis>10</wis>
+		<cha>5</cha>
+		<resist>cold, fire</resist>
+		<immune>lightning, poison, bludgeoning, piercing, and slashing from nonmagical weapons that aren't adamantine</immune>
+		<conditionImmune>charmed, exhaustion, frightened, paralyzed, petrified, poisoned</conditionImmune>
+		<senses>darkvision 60 ft.</senses>
+		<passive>10</passive>
+		<languages>understands the languages of its creator but can't speak</languages>
+		<cr>8</cr>
+		<trait>
+			<name>Berserk</name>
+			<text>Whenever the golem starts its turn with 40 hit points or fewer, roll a d6. On a 6, the golem goes berserk. On each of its turns while berserk, the golem attacks the nearest creature it can see. If no creature is near enough to move to and attack, the golem attacks an object, with preference for an object smaller than itself. Once the golem goes berserk, it continues to do so until it is destroyed or regains all its hit points.
+The golem's creator, if within 60 feet of the berserk golem, can try to calm it by speaking firmly and persuasively. The golem must be able to hear its creator, who must take an action to make a DC 15 Charisma (Persuasion) check. If the check succeeds, the golem ceases being berserk. If it takes damage while still at 40 hit points or fewer, the golem might go berserk again.</text>
```

Comment:
```suggestion
			<text>Whenever the golem starts its turn with 40 hit points or fewer, roll a d6. On a 6, the golem goes berserk. On each of its turns while berserk, the golem attacks the nearest creature it can see. If no creature is near enough to move to and attack, the golem attacks an object, with preference for an object smaller than itself. Once the golem goes berserk, it continues to do so until it is destroyed or regains all its hit points.</text>
			<text>The golem's creator, if within 60 feet of the berserk golem, can try to calm it by speaking firmly and persuasively. The golem must be able to hear its creator, who must take an action to make a DC 15 Charisma (Persuasion) check. If the check succeeds, the golem ceases being berserk. If it takes damage while still at 40 hit points or fewer, the golem might go berserk again.</text>
```